paper-server/src/main/java/io/papermc/paper/adventure/AdventureCodecs.java

```diff
@@ -228,8 +233,9 @@ public String getSerializedName() {
             CLICK_EVENT_CODEC.optionalFieldOf("clickEvent").forGetter(nullableGetter(Style::clickEvent)),
             HOVER_EVENT_CODEC.optionalFieldOf("hoverEvent").forGetter(nullableGetter(Style::hoverEvent)),
             Codec.STRING.optionalFieldOf("insertion").forGetter(nullableGetter(Style::insertion)),
-            KEY_CODEC.optionalFieldOf("font").forGetter(nullableGetter(Style::font))
-        ).apply(instance, (textColor, bold, italic, underlined, strikethrough, obfuscated, clickEvent, hoverEvent, insertion, font) -> {
+            KEY_CODEC.optionalFieldOf("font").forGetter(nullableGetter(Style::font)),
+            SHADOW_COLOR_CODEC.optionalFieldOf("shadow_color").forGetter(nullableGetter(Style::shadowColor))
```

Comment:
I'd like to mirror the order of net.minecraft.network.chat.Style.Serializer here. Move the shadow_color up under the color. Same below for application.